firefox_services_tests/apps/base.py

```diff
@@ -0,0 +1,79 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import expected, Wait
+from marionette_driver.errors import NoSuchElementException
+
+
+class Base(object):
+
+    def __init__(self, marionette):
+        self.marionette = marionette
+        self.marionette.timeout = 10000
+        self.set_context(self.marionette.CONTEXT_CONTENT)
+
+    def launch(self, url=None):
```

Comment:
why is this `None`? Navigating to `null`/`None` will make Marionette sad. I suggest perhaps making sure that it is fully formed or at least `not None`